lib/celluloid/autostart.rb

```diff
@@ -1,5 +1,6 @@
 require "celluloid"
 
+# start up the basic Celluloid services - only necessary to do this manually in old versions
```

Comment:
do you mean that the phrasing is unclear, or are you asking me to document exactly which version it changed at?